apps/ai-content-generator/src/components/config/config-section/ConfigSection.tsx

```diff
@@ -10,27 +10,17 @@ interface Props {
   model: string;
   dispatch: Dispatch<ParameterReducer>;
   isApiKeyValid: boolean;
-  localApiKey: string;
-  onApiKeyChange: (key: string) => void;
-  validateApiKey: (key: string) => Promise<boolean>;
 }
 
 const ConfigSection = (props: Props) => {
-  const { apiKey, model, dispatch, isApiKeyValid, localApiKey, onApiKeyChange, validateApiKey } =
-    props;
+  const { apiKey, model, dispatch, isApiKeyValid } = props;
 
   return (
     <Flex flexDirection="column" alignItems="flex-start" fullWidth={true}>
       <Subheading>{Sections.configHeading}</Subheading>
       <Box>
         <Form>
-          <APIKey
-            apiKey={apiKey}
-            isInvalid={!isApiKeyValid}
-            localApiKey={localApiKey}
-            onApiKeyChange={onApiKeyChange}
-            validateApiKey={validateApiKey}
-          />
+          <APIKey apiKey={apiKey} isInvalid={!isApiKeyValid} dispatch={dispatch} />
```

Comment:
`!isApiKeyValid` will negate the boolean value returned by `isApiKeyValid` right? `isApiKeyValid` returns true, then `!isApiKeyValid` will be false, and if `isApiKeyValid` returns false, then `!isApiKeyValid` will be true, 🤔 does that work as intended?

Reply:
I'm gonna rename this for readability since we're always flipping it